config/environment.js

```diff
@@ -8,8 +8,8 @@ module.exports = function(environment) {
                              'media-src': "http://datafruits.streampusher.com:8000 http://relay.datafruits.fm:8000",
                              'script-src': "'self' http://www.youtube.com http://s.ytimg.com",
                              'child-src': "http://www.youtube.com"},
-    rootURL: '/',
-    locationType: 'auto',
+    rootURL: '',
+    locationType: 'hash',
```

Comment:
hash type is required for corber apps, I wonder if we could change this to a variable that changes based on whether we are running in corber or not?
http://corber.io/pages/frameworks/ember